apispec/run.go

```diff
@@ -153,10 +153,10 @@ func Run(args Args) error {
 	if uri := args.Out.AkitaURI; uri != nil {
 		serviceName = uri.ServiceName
 		if args.Service != "" && serviceName != args.Service {
-			return errors.Errorf("--service and --out cannot specify different services")
+			return errors.Errorf("--project and --out cannot specify different projects")
```

Comment:
This might be confusing for the user if the project name came from `--service` and they are unaware of the deprecation. I don't have any easy suggestions for improving this though. 😞